gym/envs/classic_control/acrobot.py

```diff
@@ -305,9 +298,10 @@ def derivs(x,t):
         dt2 = dt / 2.0
         y0 = yout[i]
 
-        k1 = np.asarray(derivs(y0, thist, *args, **kwargs))
```

Comment:
`thist `has been removed along with` *args, *kwargs`. @TuckerBMorgan can you please verify, if this is an intended change?